.github/workflows/python.yml

```diff
@@ -38,6 +38,9 @@ jobs:
                   token: ${{ secrets.GITHUB_TOKEN }}
             - name: Build
               run: poetry build
+            - name: Validate PyProject
+              working-directory: ./e2e/python
+              run: python -m piptools compile --upgrade pyproject.toml
```

Comment:
I think this may be causing an error in CI?
`/opt/hostedtoolcache/Python/3.10.11/x64/bin/python: No module named piptools`

Reply:
needed to install `pip-tools`, fixed now